numba/pythonapi.py

```diff
@@ -1021,6 +1021,11 @@ def from_native_value(self, val, typ):
         elif isinstance(typ, types.CharSeq):
             return self.from_native_charseq(val, typ)
 
+        elif typ == types.voidptr:
```

Comment:
I don't understand why it is necessary to adapt voidptr. All the signatures seem to involve uintp.